### DISCUSSION TITLE
Failed to connect to the database. Retrying in 10 seconds. TimeoutException after 0:00:15.000000: Future not completed

### DISCUSSION THREAD

**Discussion by namtrungle20** [serverpod/serverpod#1542] [Q&A]
![image](https://github.com/serverpod/serverpod/assets/121472249/7433a1aa-338d-46fc-8e83-64a04b332cbb)
This error is weird, I don't know how to fix it, looking forward to help

**Comment by Isakdl** [comment]
Did you recreate a project with the same name?

If so the old volume of the docker container may still be on disk.

Running `docker compose down -v` will remove the old database including the data.

Then you can run `docker compose up` again to get a new instance running, make sure to also initialize the database again.
